### PR TITLE
Normalisation of mongodb workflow collection and changes in gql mutations and queries

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj Babu Das <raj.das@mayadata.io>

## Proposed changes

Changes:
1. Normalising workflow-collections and workflow-run collections.
2. Changes in getWorkflowruns gql API
3. Changes in  WorkFlowRunHandler gql API

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Comment by rajdas98** [comment]
Selection_016

File: litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go [hunk 1]
```diff
@@ -151,6 +156,7 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}
 
 	workflow_id := utils.RandomString(32)
```
Comment (gdsoumya) [inline litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go:158]: Why are we using RandomString for this? I think we should use uuid. It has a much less chance of colliding.

**Review by gdsoumya — CHANGES_REQUESTED** [review]
LGTM just need to update workflow_if to uuid

File: litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go:158]: Yes, make sense. Will update it

Comment (gdsoumya) [reply litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go:158]: Also we are not doing the project collection normalization?

**Comment by gdsoumya** [comment]
Can you resolve the conflict, the worflow_id mapping is missing from this PR.

File: litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go:158]: The project collection normalization PR will go in a separate PR.

**Review by gdsoumya — APPROVED** [review]
LGTM

**Pull request merged**